src/main.go

```diff
@@ -58,6 +50,19 @@ func main() {
 		}
 	}
 
+	// Update mechanism
+	update := make(chan struct{}, 1)
+	if len(args) > 0 && args[0] == "version" {
```

Comment:
If the command was `--version`, then we sent a [signal](https://github.com/sloppyio/cli/blob/59a0316cac1e2fb3a427aa9b52c39af7f568105a/src/main.go#L56) that cli does not have to [wait](https://github.com/sloppyio/cli/blob/59a0316cac1e2fb3a427aa9b52c39af7f568105a/src/main.go#L87) for the concurrent update process to finish because both are the same functions.